spec/ciFullConf.js

```diff
@@ -22,17 +22,17 @@ exports.config = {
     'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
     'build': process.env.TRAVIS_BUILD_NUMBER,
     'name': 'Protractor suite tests',
-    'version': '54',
-    'selenium-version': '2.53.1',
+    'version': '55',
+    'selenium-version': '3.0.1',
     'chromedriver-version': '2.25',
     'platform': 'OS X 10.9'
   }, {
     'browserName': 'firefox',
     'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
     'build': process.env.TRAVIS_BUILD_NUMBER,
     'name': 'Protractor suite tests',
-    'version': '47',
-    'selenium-version': '2.53.1'
+    'version': '50',
+    'selenium-version': '3.0.1'
```

Comment:
I'm assuming this is the selenium standalone server version. If we upgrade to selenium standalone server 3.0.x, Firefox 47 tests will not work. Can we get our test to work with FF 48+?

Also, since this would test sauce labs, the most that they support for selenium standalone server 2.53.1 and FF 47. See https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options

If this does work on saucelabs with selenium standalone server 3.0.x and FF 50, I'll need to see how this work locally.